taxonomy-editor-frontend/src/pages/allentries/index.jsx

```diff
@@ -139,16 +138,14 @@ const Entry = () => {
                 </Stack>
                 <Stack direction="row" alignItems="center" sx={{m: 2}}>
                     <Typography>Main Language</Typography>
-                    <TextField
-                        onChange={(e) => { 
-                            setNewLanguageCode(e.target.value);
-                            setIsValidLanguageCode(ISO6391.validate(e.target.value));
+                    <Autocomplete
+                        options={Object.keys(ISO_CODES)}

```

Comment:
think we don't  need to have the `ISO_CODES` const. Please check the [iso-639-1 package docs](https://www.npmjs.com/package/iso-639-1), we can obtain the names and extract the codes with it.